fluentd/Dockerfile

```diff
@@ -18,31 +18,31 @@ ENV DATA_VERSION=1.6.0 \
     PATH=/opt/app-root/src/bin:/opt/app-root/bin:/usr/libexec/fluentd/bin:$PATH \
     RUBY_VERSION=2.0
 
-ARG TEST_REPO
-
 LABEL io.k8s.description="Fluentd container for collecting of docker container logs" \
-      io.k8s.display-name="Fluentd ${FLUENTD_VERSION}" \
+      io.k8s.display-name="Fluentd" \
```

Comment:
Why ${FLUENTD_VERSION} is removed from Dockerfile but kept in Dockerfile,centos7?